src/BinSkim.Rules/DwarfRules/BA4002.ReportDwarfCompilerData.cs

```diff
@@ -91,15 +91,7 @@ private void PrintCompilerData(BinaryAnalyzerContext context, string language, I
 
                     processedRecords.Add(currentRecord);
 
-                    Console.Write($"{context.TargetUri.LocalPath},");
-                    Console.Write($"{compiler.Compiler},");
-                    Console.Write($"{compiler.Version},");
-                    Console.Write($"{compiler.Version},");
-                    Console.Write($"{language},");
-                    Console.Write($"{file},");
-                    Console.Write($"{file},");
-                    Console.Write($"{context?.Hashes?.Sha256},");
-                    Console.WriteLine();
+                    context.CompilerDataLogger.Write(compiler.Compiler.ToString(), compiler.Version.ToString(), language, file);
```

Comment:
>CompilerDataLogger [](http://example.com/codeflow?start=28&length=18)

why is it ok that some data has gotten dropped? such as the sha256 has?

Reply:
where I create the CompilerDataLogger, i pass the context. from it I retrieve information such as hash. With that, i can remove from here and just pass the data that i dont have during construction of compilerDataLogger